reference/forms/types/options/date_input_format_description.rst.inc

```diff
@@ -0,0 +1,8 @@
+If the ``input`` option is set to ``string``, this option specifies the format
+of the date. This must be a valid `PHP date format`_.
+
+.. _`PHP date format`: https://secure.php.net/manual/en/function.date.php
+
+.. versionadded:: 4.3
```

Comment:
Sorry if that gives you extra work, but I think we should move this below the `**type**` line everywhere. If there are other places where don't have it like that we should IMO update them in different PRs.

Reply:
Done.